src/samples/GroupChannelLocalCaching.js

```diff
@@ -0,0 +1,703 @@
+                    type='file'
+                    hidden={true}
+                    onChange={onFileInputChange}
+                    onClick={() => { }}
+                />
+            </div>
+
+        </div>);
+}
+
+const MembersSelect = ({
+    applicationUsers,
+    groupChannelMembers,
+    currentlyJoinedChannel,
+    addToChannelMembersList,
+    handleCreateChannel,
+    handleUpdateChannelMembersList
+
+}) => {
+
+    if (applicationUsers.length > 0) {
+        return <div className="overlay">
+            <div className="overlay-content">
+                <button onClick={() => {
+                    if (currentlyJoinedChannel) {
+                        handleUpdateChannelMembersList();
+                    } else {
+                        handleCreateChannel();
+
+                    }
+                }}>{currentlyJoinedChannel ? 'Submit' : 'Create'}</button>
+                {applicationUsers.map((user) => {
+                    const userSelected = groupChannelMembers.some((member) => member === user.userId);
+                    return <div
+                        key={user.userId}
+                        className={`member-item ${userSelected ? 'member-selected' : ''}`}
+                        onClick={() => addToChannelMembersList(user.userId)}>
+                        <ProfileImage user={user} />
+                        <div className="member-item-name">{user.nickname}</div>
+                    </div>
+                })}
+
+            </div>
+        </div >;
+    }
+    return null;
+}
+
+const CreateUserForm = ({
+    setupUser,
+    settingUpUser,
+    userNameInputValue,
+    userIdInputValue,
+    onUserNameInputChange,
+    onUserIdInputChange
+}) => {
+    if (settingUpUser) {
+        return <div className="overlay">
+            <div className="overlay-content">
+                <div>User ID</div>
+
+                <input
+                    onChange={onUserIdInputChange}
+                    className="form-input"
+                    type="text" value={userIdInputValue} />
+
+
+                <div>User Nickname</div>
+                <input
+                    onChange={onUserNameInputChange}
+                    className="form-input"
+                    type="text" value={userNameInputValue} />
+
+                <button
+                    className="user-submit-button"
+                    onClick={setupUser}>Connect</button>
+            </div>
+        </div>
+
+
+    } else {
+        return null;
+    }
+
+}
+
+const loadCachedChannels = async () => {
+    const groupChannelFilter = new GroupChannelFilter();
+    groupChannelFilter.includeEmpty = true;
+
+    const collection = sb.groupChannel.createGroupChannelCollection({
+        filter: groupChannelFilter,
+        order: GroupChannelListOrder.LATEST_LAST_MESSAGE,
+    });
+
+    collection.setGroupChannelCollectionHandler({
+        onChannelsAdded: (context, channels) => {
+            console.log('onChannelsAdded')
```

Comment:
are these handlers needed if just logging out?

Reply:
yes, I have added only for logs